### PR TITLE
adjusted family description in documentation and header file

### PR DESCRIPTION
**Pull request by Ansh-Sarkar**
Signed-off-by: Ansh Sarkar <anshsark18@gmail.com>

### CODE REVIEW

**Comment by Ansh-Sarkar** [comment]
@doug-walker could you pls review this PR

**Review by doug-walker — APPROVED** [review]

**Comment by doug-walker** [comment]
Thank you @Ansh-Sarkar.   Once we get the required second approval, we will merge this.

**Comment by Ansh-Sarkar** [comment]
@doug-walker thanks for reviewing. will be waiting for the 2nd review. Hope to contribute more to this awesome repository.

File: docs/api/c_colorspace.rst
Comment (michdolan) [inline docs/api/c_colorspace.rst:17]: This file no longer exists in the repo (as of our API doc refactor).

File: docs/api/c_processor.rst
Comment (michdolan) [inline docs/api/c_processor.rst:109]: This file no longer exists in the repo (as of our API doc refactor).

**Review by michdolan — CHANGES_REQUESTED** [review]

**Comment by michdolan** [comment]
Thanks for your contribution @Ansh-Sarkar . The change looks good, but only needs to be made now to the header file. It will propagate to the doc RST automatically now. That's the source of the merge conflict in this PR. If you merge with the master branch by allowing those two files to be removed, this should be good to merge.

**Comment by michdolan** [comment]
Also, since we just overhauled the API doc build process, and since you are changing API docs by modifying the header, you will now need to update the frozen Python API RST (in `docs/api/python/frozen`) by running the following build process:

1. Make sure you have all the documentation dependencies installed on your machine. This includes Doxygen, Python 3, and the Python packages listed in `docs/requirements.txt`. 
2. Configure and build OCIO from your build directory with:
```
cmake <path to OCIO root dir>/. -DOCIO_BUILD_DOCS=ON -DOCIO_BUILD_FROZEN_DOCS=ON
make docs -j<thread count>
```
3. `git add -u` and `commit -s` the newly generated RST files to your branch, and push to your fork to update this PR.

Apologies that your PR got caught in the middle of this all changing. Let me know if you hit any issues and I can help guide you through it. Thanks again.

**Comment by Ansh-Sarkar** [comment]
@michdolan thank you for guiding. will do the necessary changes at the earliest.

**Comment by doug-walker** [comment]
@Ansh-Sarkar, our release is approaching fast, do you think you would be able to complete this PR by Friday?

**Comment by Ansh-Sarkar** [comment]
Sorry @doug-walker will not be able to deliver. Having problems building the project. I apologize for the inconvenience

**Comment by Ansh-Sarkar** [comment]
Should have informed earlier but was trying since a long time. Sorry once again

**Comment by michdolan** [comment]
Hi @Ansh-Sarkar , just checking in to see if you would like to continue working on this change? Or would you like to close the pull request? If you need help building OCIO to update the needed rst files, let us know. Thanks!

**Comment by michdolan** [comment]
Closing this PR for now due to inactivity. Please feel free to reopen if you want to continue the contribution.

**Comment by akhilsahuji** [comment]
> ```
> cmake <path to OCIO root dir>/. -DOCIO_BUILD_DOCS=ON -DOCIO_BUILD_FROZEN_DOCS=ON
> make docs -j<thread count>
> ```
When I ran the above commands I faced the following error 

```
CMake Error at CMakeLists.txt:30 (project):
  Running

   'nmake' '-?'

  failed with:

   The system cannot find the file specified


CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!
See also "C:/Users/akhil/Documents/GSoc/OpenColorIO/CMakeFiles/CMakeOutput.log".
```

@michdolan can you help me with this?

**Pull request closed without merging**
